gatsby-node.js

```diff
@@ -1,8 +1,10 @@
 const path = require(`path`)
 const mangoSlugfy = require(`@mangocorporation/mango-slugfy`)
 const { createRemoteFileNode } = require(`gatsby-source-filesystem`)
+const { curry, __, pipe, map, isEmpty, prop, flatten } = require('ramda');
 
-const paginationPath = (page, totalPages, prefix = `/`) => {
+const paginationPath = (page, totalPages, pfx) => {
+  const prefix = isEmpty(pfx) ? `/` : pfx;
   if (page === 0) {
```

Comment:
Maybe keep the result of this validation in a variable and then have a single return at the end of the function would keep more clear, I think